specs/commandline/sign.md

```diff
@@ -33,6 +33,7 @@ Flags:
   -h,  --help                       help for sign
        --id string                  key id (required if --plugin is set). This is mutually exclusive with the --key flag
   -k,  --key string                 signing key name, for a key previously added to notation's key list. This is mutually exclusive with the --id and --plugin flags
+       --oci-layout                 [Preview] sign the artifact stored in OCI image layout
```

Comment:
```suggestion
       --oci-layout                 sign artifact stored in OCI image layout
```

Reply:
The [Preview] tag should be reserved in RC.4.

Reply:
> The [Preview] label should be reserved in RC.4.

Could you share any reason behind this? IMO, if we decide a flag won't change in the future, we should not tag it as `[Preview]`.

Reply:
Same reason as Feynman explained above.